### PR TITLE
Dime.Scheduler (Certified Connector)

### PR DESCRIPTION
**Pull request by hbulens**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by hbulens** [comment]
@microsoft-github-policy-service agree [company="Dime Software"]

**Comment by hbulens** [comment]
@microsoft-github-policy-service agree company="Dime Software"

File: certified-connectors/Dime.Scheduler/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+![Dime.Scheduler](https://cdn.dimescheduler.com/dime-scheduler/Dime.Scheduler-Black.png)
+
+# Dime.Scheduler
```
Comment (Amjed-Ayoub) [inline certified-connectors/Dime.Scheduler/README.md:3]: @hbulens please change to Publisher: Dime.Scheduler.
Please follow and include all required fields from this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)

File: certified-connectors/Dime.Scheduler/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,4404 @@
...
+          "default": {
+            "$ref": "#/responses/default"
+          }
+        }
+      }
+    },
+    "/filterGroup": {
+      "post": {
+        "summary": "Add, update or remove a filter group",
+        "description": "Add, update or remove a flter group.",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Dime.Scheduler/apiDefinition.swagger.json:662]: @hbulens Please make sure that descriptions are free from grammatical errors or typos. please change "flter" to "filter".
Thank you so much.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by Amjed-Ayoub** [comment]
Hi @hbulens thank you so much for your hard work on this connector. I had very small comments. 
Please let me know if you have any questions.
We are looking forward to getting your pull request approved and merged.
Thank you so much again for working with us.

File: certified-connectors/Dime.Scheduler/apiDefinition.swagger.json (hunk 2 above)
Comment (hbulens) [reply certified-connectors/Dime.Scheduler/apiDefinition.swagger.json]: Well spotted, thanks for pointing that out. See commit [9ee7496](https://github.com/microsoft/PowerPlatformConnectors/pull/2091/commits/9ee7496977c9b66097f89e63a13bcf590814a7cb) for the change.

File: certified-connectors/Dime.Scheduler/README.md (hunk 1 above)
Comment (hbulens) [reply certified-connectors/Dime.Scheduler/README.md:3]: The docs state the following:
> "publisher" is the name of your company or organization. Provide the full company name (for example, "Contoso Corporation"). 

Hence, Dime Software should be the only correct value as it's the company that owns Dime.Scheduler, which is one of the products that we own and manage.

Comment (Amjed-Ayoub) [reply certified-connectors/Dime.Scheduler/README.md:3]: @hbulens yes you are correct. but please add the Publisher section in the readme.
Your readme file has the following.
```
# Dime.Scheduler

Dime.Scheduler, a Dime Software product, is a resource and project planning solution for the Microsoft Dynamics product suite.
```
Please add the publisher section to be 
```
# Dime.Scheduler

Dime.Scheduler, a Dime Software product, is a resource and project planning solution for the Microsoft Dynamics product suite.

#Publisher: Dime.Scheduler
```
and please add all required sections from this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md). The readme file is missing the below sections.
```
# Obtaining Credentials
# Known Issues and Limitations
# Deployment Instructions
```

I hope this helps.

Please let me know if you have any questions.

Thank you so much for your hard work with us.

Comment (hbulens) [reply certified-connectors/Dime.Scheduler/README.md:1]: Thanks for clarifying, I did not understand your remarks referred to the readme file. Should be better now with commit [3441f57](https://github.com/microsoft/PowerPlatformConnectors/pull/2091/commits/3441f5768f2b5d1996f988a81934797fc1d7b7f5).

**Review by Amjed-Ayoub — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @hbulens,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**